nfs.go

```diff
@@ -93,13 +129,18 @@ func (h *NFSHandler) statusFile() *statusMeta {
 	return h.statusCache
 }
 
-type readCacheEntry struct {
+type readCacheEntryOld struct {
```

Comment:
Out of interest, why did we keep this around?

Reply:
Whoops. Deleted!